js/ui/viewSelector.js

```diff
@@ -243,6 +322,11 @@ const ViewsDisplayLayout = new Lang.Class({
         this._appDisplayActor.opacity = (1 - v) * 255;
         this._searchResultsActor.opacity = v * 255;
 
+        if (this._discoveryFeedButton !== null) {
```

Comment:
This check is inconsistent with the one at line 311. I'd use that style in both places.